presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTransactionalTable.java

```diff
@@ -97,18 +97,6 @@ public void testReadFullAcidBucketedV2()
         doTestReadFullAcid(false, BucketingType.BUCKETED_V2);
     }
 
-    @Test(groups = HIVE_TRANSACTIONAL, timeOut = TEST_TIMEOUT)
```

Comment:
@losipiuk suggested me to add product tests for bucket version v1/v2.

Reply:
i am aware. but they were not correct